decidim-core/app/views/decidim/searches/_results.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="row small-up-1 medium-up-2 card-grid">
   <% @results.each do |result| %>
-    <%= card_for result.resource, context: { label: true, current_user: current_user }, show_footer_actions: false %>
```

Comment:
Why remove this?

Reply:
It was not having any effect even setting it to true or to false.
Also, setting any value into `:options` other than context is being ignored so I had to set :show_space flag into context.

Reply:
It is used by the `MeetingsMCell`, because the "Join" button does not work properly from the search page. Please, readd it 🙏

Reply:
Have you tested that this flag works? I'm logging its value in `meeting_m_cell#show_footer_actions?` and it is always `nil` whatever the value I set it to, `true` or `false`.

Reply:
I've just checked that the list of meetings (having meetings with registration enabled) does not render the join button either.

Reply:
@tramuntanal then that's a bug 😅 do you mind opening an issue and I'll take care of it?

Thanks!